partridge/gtfs.py

```diff
@@ -34,9 +35,13 @@ def __init__(
         self._locks: Dict[str, RLock] = {}
         if isinstance(source, self.__class__):
             self._read = source.get
+            self._proxy_feed = bool(self._view)
```

Comment:
I would prefer the value of `_proxy_feed` not to depend on whether in feed is initialized from a path or another feed object. Is that possible?

Reply:
Yes, it could probably just be `bool(self.view)` out side the of if block

Reply:
I tried that and it didn't work.
Do you prefer passing proxy as a parameter to feed.__init__ ?